model/src/test/kotlin/ScanResultContainerTest.kt

```diff
@@ -63,45 +64,46 @@ class ScanResultContainerTest : WordSpec() {
         scannerEndTime1,
         1,
         sortedSetOf(
-            LicenseFindings(
+            LicenseFinding(
                 "license 1.1",
-                sortedSetOf(TextLocation("path 1.1", 1, 1)),
-                sortedSetOf(
-                    CopyrightFindings(
-                        "copyright 1",
-                        sortedSetOf(TextLocation("copyright path 1.1", 1, 1))
-                    )
-                )
+                TextLocation("path 1.1", 1, 1)
             ),
-            LicenseFindings(
+            LicenseFinding(
                 "license 1.2",
-                sortedSetOf(TextLocation("path 1.2", 1, 2)),
-                sortedSetOf(
-                    CopyrightFindings(
-                        "copyright 2",
-                        sortedSetOf(TextLocation("copyright path 1.2", 1, 2))
-                    )
-                )
+                TextLocation("path 1.2", 1, 2)
             )
         ),
+        sortedSetOf(
+            CopyrightFinding(
+                "copyright 1",
+                TextLocation("copyright path 1.1", 1, 1)
+            ),
+            CopyrightFinding(
+                "copyright 2",
+                TextLocation("copyright path 1.2", 1, 2)
+            )
+         ),
         mutableListOf(error11, error12)
     )
     private val scanSummary2 = ScanSummary(
         scannerStartTime2,
         scannerEndTime2,
         2,
         sortedSetOf(
-            LicenseFindings(
+            LicenseFinding(
                 "license 2.1",
-                sortedSetOf(),
-                sortedSetOf(CopyrightFindings("copyright 3", sortedSetOf()))
+                TextLocation("", 1, 2)
```

Comment:
I guess this is where the empty path comes from. To match the original code we should pass an empty set here instead, I believe.

Reply:
This is nor a set neither a property.
This change requires exactly one location per finding.

Reply:
done.